src/main/steps/applicant1/hub-page/joint/form.njk

```diff
@@ -23,14 +23,13 @@
       name: applicantApplyForConditionalOrderStarted,
       value: "Yes"
     }) }}
-  {% elif displayState.state() == 'AwaitingFinalOrder' %}
-      {{ govukButton({
-        text: button.applyForFinalOrder,
-        classes: form.submit.classes,
-        type: "submit",
-        name: "TODO",
-        value: "Yes",
-        preventDoubleClick: true
-      }) }}
+    {% elif displayState.state() == 'AwaitingFinalOrder' or 'AwaitingJointFinalOrder' %}
```

Comment:
dont you already compare these in the content, maybe use them.

Reply:
This check is only for the button on the Hub page, if I remove the if statement the button doesn't show.

Reply:
the comment above where you compare the states is pretty much the same as here but its fine.
have you tested this for AwaitingJointFinalOrder
because I dont think that or part is going to work as you expect it to.

Reply:
@velimd , yes I tested it when the state is Awaiting joint final order, the button does show up on the hub page and afterwards it takes you to the finalising-your-application page. I've attached a screenshot in the ticket. https://tools.hmcts.net/jira/browse/NFDIV-1607